src/libraries/System.Private.Uri/src/System/Uri.cs

```diff
@@ -3262,16 +3262,17 @@ private unsafe void ParseRemaining()
             // For iri parsing if we found unicode the idx has offset into _originalUnicodeString..
             // so restart parsing from there and make _info.Offset.Path as _string.Length
 
-            idx = _info.Offset.Path;
-            origIdx = _info.Offset.Path;
+            idx = origIdx = _info.Offset.Path;
```

Comment:
Why this change?

Reply:
I find this way makes it more obvious that the same value is assigned to both variables.

I can revert this as it has no impact otherwise.